tests/framework/CodeInterfaceTests/Serpent/SerpentTest1/sampleInput/2/out~testInput

```diff
@@ -0,0 +1,5828 @@
+
+  _                   .-=-.           .-=-.          .-==-.       
```

Comment:
This file is the logfile (screen output). It is not functional for testing the interface. Please remove it